test/providers.tf

```diff
@@ -7,9 +7,8 @@ variable "google_region" {
 }
 
 provider "google" {
-
   // The google_service_account module uses `project`.
-  project     = var.project
+  project     = var.project_id // Defined in examples
```

Comment:
not sure what the comment means here

Reply:
<img width="619" alt="Screen Shot 2022-08-30 at 4 18 23 PM" src="https://user-images.githubusercontent.com/3286150/187399588-58ba2c9b-b821-4186-8498-e3243c9c0f61.png">

IDE isn't happy if you look at this file in isolation, since it is copied at runtime.